### PR TITLE
added support for global labels

### LINKED ISSUE #459: Add support for global labels
See https://github.com/elastic/apm-agent-go/issues/507

### PR DESCRIPTION
**Pull request by beniwohli**
fixes #459

### CODE REVIEW

File: elasticapm/conf/__init__.py [hunk 1]
```diff
@@ -101,6 +101,22 @@ def __set__(self, instance, value):
...
+class _DictConfigValue(_ConfigValue):
+    def __init__(self, dict_key, item_separator=",", keyval_separator="=", **kwargs):
+        self.item_separator = item_separator
+        self.keyval_separator = keyval_separator
+        super(_DictConfigValue, self).__init__(dict_key, **kwargs)
+
+    def __set__(self, instance, value):
+        if isinstance(value, compat.string_types):
+            items = (item.split(self.keyval_separator) for item in value.split(self.item_separator))
+            value = {key: self.type(val) for key, val in items}
```
Comment (axw) [inline elasticapm/conf/__init__.py]: I think we should strip key and val. Someone's bound to have white space in their environment variables.

File: elasticapm/conf/__init__.py [hunk 2]
```diff
@@ -101,6 +101,22 @@ def __set__(self, instance, value):
...
+        elif not isinstance(value, dict):
+            # TODO: better error handling
+            value = None
+        instance._values[self.dict_key] = value
```
Comment (axw) [inline elasticapm/conf/__init__.py]: should probably check that the tag keys match `TAG_RE` here? also, truncate value to 1024 chars?

File: elasticapm/conf/__init__.py (hunk 1 above)
Comment (beniwohli) [reply elasticapm/conf/__init__.py]: done in 06b64dc9d3ec

File: elasticapm/conf/__init__.py (hunk 2 above)
Comment (beniwohli) [reply elasticapm/conf/__init__.py]: Both done in 06b64dc9d3ec

**Pull request closed without merging**